pkgs/by-name/ca/cargo-aoc/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  stdenv,
+  darwin,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-aoc";
+  version = "0.3.7";
+
+  src = fetchFromGitHub {
+    owner = "gobanos";
+    repo = "cargo-aoc";
+    rev = version;
+    hash = "sha256-k9Lm91+Bk6EC8+KfEXhSs4ki385prZ6Vbs6W+18aZSI=";
+  };
+
+  cargoHash = "sha256-DKP9YMbVojK7w5pkX/gok4PG6WUjhqUdvTwSir05d0s=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [
```

Comment:
this shouldn't need to be included with the new darwin SDK pattern. See https://github.com/NixOS/nixpkgs/issues/354146, specifically the first migration example.

Reply:
Thanks!